packages/math-input/src/components/input/math-wrapper.ts

```diff
@@ -211,12 +215,12 @@ class MathWrapper {
             this._handleBackspace(cursor);
         } else if (key === Keys.LEFT) {
             this._handleLeftArrow(cursor);
-        } else if (key === Keys.RIGHT || key === Keys.JUMP_OUT) {
+        } else if (key === Keys.RIGHT) {
```

Comment:
Was this deletion intentional?

Reply:
Yep, couldn't find any other reference to `JUMP_OUT` and it was throwing an error when I typed `key`